lib/create-response-handler.ts

```diff
@@ -17,9 +17,11 @@ import { TranslatedMatchingResponseBody, TranslatedIdentityResponseBody, Scenari
  * ResponseScenarios
  *
  * An object enumerating scenarios that can occur when handling a response from
- * Verify.
+ * Verify when a matching service adapter is used.
```

Comment:
```suggestion
 * GOV.UK Verify Hub.
```
I removed `when a matching service adapter is used` assuming the sentence from `NOTE` below will be moved up before this sentence.